nautobot/core/views/mixins.py

```diff
@@ -909,7 +912,41 @@ def perform_update(self, request, *args, **kwargs):  # pylint: disable=arguments
             return self.form_invalid(form)
 
 
-class ObjectBulkDestroyViewMixin(NautobotViewSetMixin, BulkDestroyModelMixin):
+class DeleteAllModelMixin:
+    """
+    UI mixin to bulk destroy all model instances.
+    """
+
+    def _get_bulk_delete_all_queryset(self, request):
+        model = self.queryset.model
+
+        # This Mixin is currently been used by both NautobotViewSet ObjectBulkDestroyViewMixin
+        # and BulkDeleteView which uses different keys for accessing filterset
+        filterset = getattr(self, "filterset", None)
```

Comment:
Nit: I'd prefer `filterset_class` as the variable name in this function rather than `filterset`.